### ISSUE TITLE
Add social icons

### ISSUE THREAD

**Issue by mattleff** [SimpleUpdates/SUThemeALPS#3]

**Comment by su-narthur** [comment]
@mattleff So we're abandoning ALPS' text links?

**Comment by mattleff** [comment]
@su-narthur No, this is two different things. You're talking about the links that link to the social pages of the site (like `https://www.facebook.com/matthew.leffler`). I'm talking about the social icons to share the current page (like `https://www.facebook.com/share.php?u=http://answersforme.org/`).

**Comment by su-narthur** [comment]
Ah, yes. Alps does provide [this component](http://alps.adventist.io/public/patterns/01-molecules-components-share-tools/01-molecules-components-share-tools.html) if we'd like to go that route.

**Comment by mattleff** [comment]
Yeah, I'm good with that.
